src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -180,6 +180,7 @@ def load_arguments(self, _):
         c.argument('addons', options_list=['--addons', '-a'], validator=validate_addons)
         c.argument('subnet_name', options_list=['--subnet-name', '-s'])
         c.argument('disable_sgxquotehelper', action='store_true')
+        c.argument('osm_mesh_name', options_list=['--osm-mesh-name'])
```

Comment:
Is there any validation we can do on the input param? For example, on name length or allowed chars etc?

Reply:
Discussed this with Robbie, for a start we prefer to keep the validations in the backend. We are still discussing some design considerations and might even consider removing this option in the near term.

Reply:
Ok, if you keep it then please add the validator before GA